### ISSUE TITLE
Installation/Usage problems

### ISSUE THREAD

**Issue by MichaelDBA** [postgres-ai/postgres-checkup#154]
I understand that the only way to use this is to do a git clone download, but it should be more straightforward in how to get it working.

It is not a simple as copying a binary file to a normal bin location like /usr/bin or /usr/local/bin.  The program requires other files in other subdirectories, ie. resources, in the downloaded project.

Can you offer some suggestions?  Or do I always have to run it from the git cloned subdirectory?

**Comment by binakot** [comment]
Hello, @MichaelDBA!

Atm I'm trying to contribute Docker environment for tool ;) 
More info here: https://gitlab.com/postgres-ai-team/postgres-checkup/merge_requests/274.
When it will be done, Postgres.ai deploy ready-to-use image on Docker Hub or somewhere else.

Is it enough for your workflow?

**Comment by MichaelDBA** [comment]
Not really, I will put a bandaid on it until it gets more easy to use outside of docker.
